layers/+emacs/org/packages.el

```diff
@@ -588,6 +590,19 @@ Headline^^            Visit entry^^               Filter^^                    Da
       (kbd "M-SPC") 'spacemacs/org-agenda-transient-state/body
       (kbd "s-M-SPC") 'spacemacs/org-agenda-transient-state/body)))
 
+(defun org/init-org-notify ()
+  (use-package org-notify
+    :config
```

Comment:
Can we lazy load the package?

Reply:
Ah yes, we could lazy load it. I first loaded it `:after` org-agenda, but then I figured that users would like the notification daemon to start at Spacemacs startup. But the `org-notify-start` function is autoloadable so yes, we could lazy load it (althoguh it would still start at startup if users choose to start the daemon at startup).

Reply:
Are you sure, the config code is only executed when the package is loaded, so if you call org-notifiy-start in there and lazy the autoloading on that function will not trigger it. Instead you would need to load the server in the init part, if requested.

Reply:
Yes it should be moved too :init then also :+1:

Reply:
Ah okay, it looks like `org-notify-start` is not autoloadable. So I think we should keep it as it is now. Do you think I should create a PR for making it autoloadable?

Reply:
I think you can declare it autoloadable in use-package directly using the `:commands` tag. Still an up-stream PR would be nice for the rest of the emacs world.

Reply:
Ah great! I did not know that feature yet